packages/react/src/LogoSuite/LogoSuite.tsx

```diff
@@ -235,7 +247,12 @@ const _LogoBar = forwardRef(
 
     const defaultProps = {
       ref,
-      className: clsx(styles['LogoSuite__logobar'], styles[`LogoSuite__logobar--variant-${variant}`], className),
+      className: clsx(
+        styles['LogoSuite__logobar'],
+        styles[`LogoSuite__logobar--variant-${variant}`],
+        styles[`LogoSuite__logobar--gap-${gap}`],
```

Comment:
@danielguillan Heads up that this rule isn't applying due to low specificity in marquee mode. 

<img width="573" alt="Image" src="https://github.com/user-attachments/assets/435e82b5-bd2c-46a5-b41f-7a390b8d4927" />

To resolve the original request, we need this to work in marquee mode so could we please:

1. Increase specificity of the gap rule
2. Update the gap story to show a marquee instead (or maybe both)